includes/ajax.php

```diff
@@ -334,301 +334,28 @@ function tsml_ajax_geocode()
 
 // ajax function to import the meetings in the import buffer
 // used by admin_import.php
-add_action('wp_ajax_tsml_import', function () {
-    global $tsml_data_sources, $tsml_custom_meeting_fields, $tsml_source_fields_map, $tsml_contact_fields, $tsml_entity_fields, $tsml_array_fields;
```

Comment:
This removed block of logic, responsible for importing meetings from the buffer, is moved to `tsml_import_buffer_next( $count = 25 )`